src/adapters/bidderFactory.js

```diff
@@ -250,6 +255,17 @@ export function newBidder(spec) {
     }
   });
 
+  function isUnknownBidder(unknownBidder, bidderCode) {
```

Comment:
Also, can I suggest different parameter names here - `requestBidder` and `responseBidder` instead of `unknownBidder` and `bidderCode`?

Reply:
Hi @dgirardi,
Thank you for your suggestion. I have made the changes. :)